app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,14 @@
 # frozen_string_literal: true
 
-# see observer_controller.rb
-module ObserverController::MarkupController
+class LookupsController < ApplicationController
+  # These need to be moved into the files where they are actually used.
+  require "find"
+  require "set"
+
+  before_action :login_required, except: [
+    :lookup_observation,
+  ]
+
```

Comment:
Very minor point, but I bet you'll agree -- the "lookup" in the action name is now redundant, right?

Reply:
Agree. Seemed a bit... bare... without it, but i'll just go for it. These actions are namespaced under the class, so no one's going to get confused. Hopefully.

Reply:
Not feeling like changing these... too confusing without the prefix. 
But did you want to remove `lookup_observation`?